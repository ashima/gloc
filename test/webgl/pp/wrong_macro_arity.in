#define G() 5
#define F(A) (A)
void main() {
     F((3,2));
     F();
     F(5,3);
     G();
     G(3);
     G(3,4);
}
