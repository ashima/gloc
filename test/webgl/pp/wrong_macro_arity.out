File 0, line 6, col 5 - 6:
too many macro arguments: expected 1, got 2

File 0, line 8, col 5 - 6:
too many macro arguments: expected 0, got 1

File 0, line 9, col 5 - 6:
too many macro arguments: expected 0, got 2

Fatal: unrecoverable preprocessor error (4)
